src/apps/material/material.dev.tsx

```diff
@@ -26,6 +26,16 @@ export default {
       defaultValue: "1",
       control: { type: "text" }
     },
+    blacklistedBranchesConfig: {
+      name: "Blacklisted branches",
+      defaultValue: "FBS-751032,FBS-751031,FBS-751009,FBS-751027,FBS-751024",
+      control: { type: "text" }
+    },
```

Comment:
Please add seperate configuration for pickup and availability branches.

There are actually 3 different configurations for the different parts of the system that needs to be blacklisted.